software/controller/lib/hal/hal_stm32.cpp

```diff
@@ -269,9 +269,6 @@ void HalApi::InitUARTs() {
   //        Need to do that as soon as the boards are available.
   enable_peripheral_clock(PeripheralID::USART2);
   enable_peripheral_clock(PeripheralID::USART3);
-#if defined(UART_VIA_DMA)
```

Comment:
why is this no longer necessary?

Reply:
because the DMA clocks are now enabled in DMA::ChannelControl::Initialize()